src/rpc/rawtransaction.cpp

```diff
@@ -1484,7 +1484,7 @@ UniValue createpsbt(const JSONRPCRequest& request)
                                 {
                                     {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The transaction id"},
                                     {"vout", RPCArg::Type::NUM, RPCArg::Optional::NO, "The output number"},
-                                    {"sequence", RPCArg::Type::NUM, /* default */ "", "The sequence number"},
+                                    {"sequence", RPCArg::Type::NUM, /* default */ "depends on the value of the 'replaceable' and 'locktime' arguments", "The sequence number"},
```

Comment:
14877: we have no `replaceable`

Reply:
hmm... you dropped it all for some reason but we do have `locktime` and it does affect `sequence`, should rather be `depends on the value of the 'locktime' argument` imo.

Reply:
Hello @UdjinM6 I have corrected it now